validation/api/api.lock.json

```diff
@@ -898,11 +898,13 @@
   {
     "name": "Random",
     "exports": [
-      "public class AsyncRandom(generator : () -> async* Blob)",
+      "public class AsyncRandom(",
```

Comment:
Weird truncation, probably coze args don't fit on a line...

Reply:
Thanks for pointing that out; it's for the reason you mentioned as well as the regex-based logic not being able to handle the open/close curly braces + nested parentheses in the record types. I'll address this in a separate PR, since this overlaps with the logic needed for the code coverage checker.